tests/access_tests.py

```diff
@@ -370,7 +373,10 @@ def test_update_role(self):
         resp = self.client.post(
             '/superset/update_role/',
             data=json.dumps({
-                'usernames': ['gamma'],
+                'users': [{
+                        'username': 'gamma',
+                        'email': 'gamma@superset.com'
```

Comment:
I've added the email field, actually we plan to send emails for the access requests )